apps/protocol-frontend/src/components/AttestationsTable.tsx

```diff
@@ -135,18 +135,21 @@ const AttestationsTable = ({
 
           const guildHasVerificationFramework =
             row.original.guilds[0].guild?.verification_setting_id !== null;
+          const currentAttestations = row.original.attestations?.length;
           const attestationThreshold =
             row.original.guilds[0].attestation_threshold;
-          const currentAttestations = row.original.attestations?.length;
           const frameworkSettingThreshold =
             row.original.guilds[0].guild.verification_setting
               ?.num_of_attestations;
           let pillUnverifiedLabel!: string;
           if (attestationThreshold && frameworkSettingThreshold) {
-            pillUnverifiedLabel = `${
-              frameworkSettingThreshold - currentAttestations
-            }/${frameworkSettingThreshold}`;
+            pillUnverifiedLabel = `${currentAttestations}/${frameworkSettingThreshold}`;
```

Comment:
sweet

Reply:
I know you caught that yesterday but I didn't remove the frameworkThreshold from the first part so it wasn't accurate!